packages/grpc-js/src/server.ts

```diff
@@ -640,6 +640,57 @@ export class Server {
 
       this.sessions.add(session);
 
+      // TODO: Maybe also implement support for grpc.keepalive_permit_without_calls?
+
+      if ('grpc.keepalive_time_ms' in this.options) {
```

Comment:
nit: Just do the lookup once vs the double lookup and you can avoid all the '!' etc.

```
const keepAliveTimeMs = this.options['grpc.keepalive_time_ms'];
if (keepAliveTimeMs) {
...
}
```